server/api/reports.py

```diff
@@ -479,27 +479,35 @@ def round_rows(election: Election):
                 .filter(SampledBatchDraw.ticket_number != EXTRA_TICKET_NUMBER)
                 .all()
             )
+            sampled_batch_ids = {batch.id for batch in sampled_batches}
             combined_sub_batches = (
                 Batch.query.join(Jurisdiction)
                 .filter_by(election_id=election.id)
                 .filter(Batch.combined_batch_name.isnot(None))
                 .all()
             )
-            combined_batches = group_combined_batches(combined_sub_batches)
-
-            num_sampled_batches = (
-                len(sampled_batches)
-                # Instead of counting sub batches that got combined, count the
-                # combined batches
-                - len(combined_sub_batches)
-                + len(combined_batches)
```

Comment:
This was wrong in a few ways:
- It counted combined batches that had no RLA sampled batches in them (e.g. combined batches with only extra sampled batches and unsampled batches)
- It subtracted unsampled sub batches (that are included in `len(combined_sub_batches)`) rather than only the sampled sub batches

You can see the impact of the latter in the test snapshot update for test_batch_comparison.py, which has nothing to do with extra batch sampling.

Reply:
Thanks for the clarification!